### PR TITLE
Sel3

### PR DESCRIPTION
**Pull request by RutvikChandla**

### CODE REVIEW

**Comment by asambstack** [comment]
Please add .DS_Store to gitignore

**Comment by RutvikChandla** [comment]
> Please add .DS_Store to gitignore

Done.

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
+# csharp-selenium-browserstack
 
+## Prerequisite
+---
```
Comment (asambstack) [inline README.md]: Remove the divider

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
+<b>Dotnet CLI</b> must be installed on your system. Check the installation exists before running script.
+
+```
+dotnet --version
+```
+## Steps to run test sessions
+---
```
Comment (asambstack) [inline README.md]: Remove the divider

File: README.md [hunk 3]
```diff
@@ -1 +1,74 @@
...
+### CLone and build the project
```
Comment (asambstack) [inline README.md]: Typo

File: README.md [hunk 4]
```diff
@@ -1 +1,74 @@
...
+```
+git clone https://github.com/browserstack/csharp-selenium-browserstack.git
+cd csharp-selenium-browserstack
+dotnet build
+```
+### Configure Credentials in each test files (SingleTest.cs, LocalTest.cs, Paralleltest.cs)
```
Comment (asambstack) [inline README.md]:
```suggestion
### Configure credentials in each test files (SingleTest.cs, LocalTest.cs, ParallelTest.cs)
```

File: README.md [hunk 5]
```diff
@@ -1 +1,74 @@
...
+```c#
+ // Update your credentials
```
Comment (asambstack) [inline README.md]: Remove this comment

File: README.md [hunk 6]
```diff
@@ -1 +1,74 @@
...
+String BROWSERSTACK_USERNAME = "BROWSERSTACK_USERNAME";
+String BROWSERSTACK_ACCESS_KEY = "BROWSERSTACK_ACCESS_KEY";
+```
+---
+### Run Sigle Test
```
Comment (asambstack) [inline README.md]:
```suggestion
### Run Single Test
```

File: README.md [hunk 7]
```diff
@@ -1 +1,74 @@
...
+capability.AddAdditionalCapability("realMobile", "true");
+capability.AddAdditionalCapability("os_version", "14.0");
+capability.AddAdditionalCapability("name", "BStack-[C_sharp] Sample Test"); // test name
+capability.AddAdditionalCapability("build", "BStack Build Number 1"); // CI/CD job or build name
+```
+iii. Run your test <br/>
+```
+dotnet run Program.cs single
+```
+---
```
Comment (asambstack) [inline README.md]: Remove the divider

File: README.md [hunk 8]
```diff
@@ -1 +1,74 @@
...
+capability.AddAdditionalCapability("realMobile", "true");
+capability.AddAdditionalCapability("os_version", "14.0");
+capability.AddAdditionalCapability("name", "BStack-[C_sharp] Sample Test"); // test name
+capability.AddAdditionalCapability("build", "BStack Build Number 1"); // CI/CD job or build name
+```
+iii. Run your test <br/>
+```
+dotnet run Program.cs local
+```
+---
```
Comment (asambstack) [inline README.md]: Remove the divider

File: README.md [hunk 9]
```diff
@@ -1 +1,74 @@
...
+### Run Parallel Test
```
Comment (asambstack) [inline README.md:53]: This isn't how we run the `ParallelTest.cs` file, what does configure capabilities mean in this context? Please mention the right steps on how to configure multiple tests. Also, please maintain parity between the build names mentioned in readme and the actual scripts.

**Review by asambstack — CHANGES_REQUESTED** [review]

**Comment by asambstack** [comment]
Please make sure we do not leave any typos, inconsistencies or dangling divider lines in the readme of browserstack's official code repos. 
Also, review the changes once before requesting external reviews. I can still see the .DS_Store file hasn't been deleted after last time I reviewed

**Review by asambstack — APPROVED** [review]

**Pull request merged**